README.md

```diff
@@ -1,18 +1,16 @@
 # syrupy
 
-<span><img align="right" width="200" height="200" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo"></span>
+<img width="100px" height="100px" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo">
```

Comment:
I just think it looks better right aligned

<img width="917" alt="Screen Shot 2019-12-26 at 11 10 37 PM" src="https://user-images.githubusercontent.com/2528959/71500907-180b6880-2835-11ea-9e5b-ca19640118e4.png">

```suggestion
<img align="right" width="100px" height="100px" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo">
```